yarn.lock

```diff
@@ -5634,12 +5634,15 @@ __metadata:
     "@metamask/eth-sig-util": "npm:^8.0.0"
     "@metamask/eth-simple-keyring": "npm:^6.0.5"
     "@metamask/keyring-api": "npm:^10.1.0"
-    "@metamask/message-manager": "npm:^11.0.1"
+    "@metamask/message-manager": "npm:^11.0.2"
     "@metamask/utils": "npm:^10.0.0"
     async-mutex: "npm:^0.5.0"
     ethereumjs-wallet: "npm:^1.0.1"
     immer: "npm:^9.0.6"
-  checksum: 10/4614b53f9dd053edcc470e1d949a57ee5982b544c3ea9c6635b6a2d40fef5f2bd606aa8a45d3a5c519e93dc60c45ba5a5efafefb2c9843c85522477335d208a2
+  peerDependencies:
+    "@metamask/providers": ^18.1.0
+    webextension-polyfill: ^0.10.0 || ^0.11.0 || ^0.12.0
```

Comment:
It was not marked as a breaking change because it was expected that clients that needed these would have needed to add these as dependencies already in order to even have a working build: https://github.com/MetaMask/core/pull/4974. Maybe that was not the right decision, but that was the thinking.

Reply:
I see that the extension is using `webextension-polyfill` 0.8.0 when it should be using at least 0.10.0. But I feel like that has never been correct and we should upgrade it when we get the chance.

Reply:
Is this pr safe to merge ?

Reply:
Peer dependencies are just suggestions — it's up to the client to provide those packages. If the extension already builds today then this upgrade should not cause any issues, but we should probably make a note to upgrade `webextension-polyfill` in the future.

Other than that, if we feel that these new peer dependencies should have been released as breaking changes, I can cut new releases to indicate that, just let me know.